docs/developer.md

```diff
@@ -0,0 +1,64 @@
+# Mobile App Developer's Guide
+
+This guide outlines the best practices to follow when you’re building a mobile app for the Province of B.C. 
+
+
+## Digital Code of Practice
+
+All apps developed for the Province of B.C. must adhere to the [Digital Code of Practice](https://digital.gov.bc.ca/policies-standards/dcop/) (DCoP). It provides a comprehensive set of guidelines and principles to guide your work. The DCoP emphasizes user-centricity, accessibility, security, and efficiency. Adhering to it will help you achieve digital excellence and improve citizen experience.
+
+## Code location
+
+[Work in the open](https://digital.gov.bc.ca/policies-standards/dcop/open/) is one of the principles of the [Digital Code of Practice](https://digital.gov.bc.ca/policies-standards/dcop/). As such, we encourage teams to host their app's code in the [bcgov GitHub organization](https://github.com/bcgov). Learn how to do this in the [Working in the BCGov GitHub organization](https://docs.developer.gov.bc.ca/start-working-in-bcgov-github-organization/) documentation.
+
+
+While we encourage working in the open, we realize there are situations where it may not be possible. If this is your case, then other code hosting options are available. The [BC Government organizations in GitHub](https://docs.developer.gov.bc.ca/bc-government-organizations-in-github/) documentation provides information on alternative locations.
```

Comment:
```suggestion
We recognize there are situations where it may not be possible to share your code publicly and other code hosting options are available. The [BC Government organizations in GitHub](https://docs.developer.gov.bc.ca/bc-government-organizations-in-github/) documentation provides information on alternative locations.
```